Chrome/manifest.json

```diff
@@ -1,15 +1,13 @@
 {
-	"manifest_version": 2,
+	"manifest_version": 3,
 	"name": "Grammar Annoyer",
-	"version": "1.0",
+	"version": "1.0.1",
 	"description": "Annoy you're friends! Makes common grammar mistakes across the web.",
-	"content_scripts": 
+	"content_scripts":
 	[
 		{
 			"matches": ["*://*/*"],
-			"js": ["content_script.js"],
-			"run_at": "document_end"
```

Comment:
Curious, as I am not a web dev, why change this?

Reply:
It appears to be merely an upgrade to the manifest v3 interface 😬